src/Assembly.js

```diff
@@ -138,8 +138,8 @@ class Assembly extends React.Component {
   @observable notes              = new Records("Note"           , "notes")
 
   // Current medication report
-  @observable survey_date = moment()
-  @observable survey_medication_time = moment()
+  @observable survey_date = moment().format("YYYY-MM-DD")
+  @observable survey_medication_time = moment().format("HH:mm")
```

Comment:
Nice. I'll try to migrate these to https://moment.github.io/luxon/ later, but for now this will be perfect.

See https://moment.github.io/luxon/docs/manual/moment.html for differences.